test/test_chunkstore.py

```diff
@@ -22,6 +22,19 @@ def setUp(self) -> None:
         )
         self._store = CciChunkStore(cci_odp, dataset_id, cube_params)
 
+    @skipIf(os.environ.get('XCUBE_DISABLE_WEB_TESTS', None) == '1', 'XCUBE_DISABLE_WEB_TESTS = 1')
+    def test_unconstrained_chunk_store(self):
+        cci_odp = CciOdp()
+        dataset_id = 'esacci.OZONE.mon.L3.NP.multi-sensor.multi-platform.MERGED.fv0002.r1'
+        store = CciChunkStore(cci_odp, dataset_id, None)
```

Comment:
```suggestion
        store = CciChunkStore(cci_odp, dataset_id, cube_params=None)
```

Just to be a little more explicit which makes reading code by humans easier.